pkg/tools/prefixpool/prefixpool_test.go

```diff
@@ -137,6 +137,14 @@ func TestReleaseExcludePrefixes(t *testing.T) {
 	require.Equal(t, []string{"10.20.0.0/16"}, pool.GetPrefixes())
 }
 
+func TestReleaseExcludePrefixesNestedNetworks(t *testing.T) {
```

Comment:
Also, we can check that if the initial prefix is 10.20.0.0/16 then nothing is changed on releasing "10.20.0.1/21", "10.20.2.1/21", "10.20.2.1/16"